common/include/Utilities/IniInterface.h

```diff
@@ -70,10 +70,11 @@ class IniInterface
 	void EnumEntry( const wxString& var, T& value, const wxChar* const* enumArray=NULL, const T defvalue=(T)0 )
 	{
 		int tstore = (int)value;
+		int defaultvalue = (int)defvalue;
```

Comment:
Well, I remember a previous discussion where you had mentioned to consider using``enum class`` over normal enums at all scenarios to prevent scope/name classes. Also personally, I prefer ``enum class`` over ``enum``. ``( Yes, I noticed the pseudo scope but I still feel enum classes are better)``

Though honestly what's wrong with the modification on ``IniInterface.h``? It should also work for other enums as they have an underlying type of ``int``.